tests/qos/test_qos_sai.py

```diff
@@ -1092,6 +1092,19 @@ def testQosSaiLossyQueueVoq(
 
         self.updateTestPortIdIp(dutConfig, get_src_dst_asic_and_duts, qosConfig[LossyVoq])
 
+	dst_port_id = dutConfig["testPorts"]["dst_port_id"]
+	dst_port_ip = dutConfig["testPorts"]["dst_port_ip"]
+        if separated_dscp_to_tc_map_on_uplink:
+            # We need to choose only the downlink port ids, which are associated
+            # with AZURE dscp_to_tc mapping. The uplink ports have a
+            # different mapping.
+            for index in range(length(dutConfig['testPorts']['downlink_port_ids']):
+                if dutConfig["testPorts"]["src_port_id"] !=
```

Comment:
```
E     File "/azp/_work/78/s/tests/qos/test_qos_sai.py", line 1110
E       if dutConfig["testPorts"]["src_port_id"] !=
E                                                 ^
E   SyntaxError: invalid syntax
```
@rraghav-cisco  wondering if you ran test for you code?

Reply:
Fixed the code. Waiting for TB availabiliety to run.